Datasets/data-connect-dataset-licensedetails.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Microsoft Graph Data Connect LicenseDetails_v0 dataset"
+description: "Use the LicenseDetails_v0 dataset to provide detailed information related to all the devices that are owned by each user in the organization."
+author: "rimisra2"
+ms.localizationpriority: high
+ms.prod: "data-connect"
+ms.custom: datasets:dataset-name
+---
+
+# LicenseDetails_v0 dataset
+
+The LicenseDetails_v0 dataset provides detailed information related to all the devices that are owned by each user in the organization.
+
+## Scenarios
+
+The following are business scenarios that you can answer with this dataset:
+
+- Analyse the licenses assigned to all users of a tenant
+- Analyze the license provisioning status of every license assigned to users and entire company
+- Analyze the security of the company's resources by confirming the correct set of licenses assigned to every user  
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- What are the licenses assigned to all the users?
+- Are all the licenses provisioned for all the users?
+- What is the status of license purchased for any users?
+- Who are the users who can access microsoft products purchased by the company? 
+
+## Joining with other datasets
+
+The LicenseDetails_v0 dataset can be joined with relevant users, and other category datasets.
+
+## Definitions
+
+- Licenses are assigned for every purchase of any microsoft product/service(eg: M365) by a tenant. They represent the users/groups allowed to use products within a company.  
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| id | string | The unique identifier of the license details object | No | None |
+| skuID | string | Unique Identifier(GUID) for the service SKU. Equal to the skuID property on the related SubscribedSKU object. | No | None |
+| skuPartNumber | string | Unique SKU display name. Equal to the skuPartNumber on the related SubscribedSku object. | No | None |
+| servicePlans | array | Information about the service plans assinged with the license. FORMAT:- ARRAY<STRUCT<'servicePlanId':'String', 'servicePlanName': 'String', 'provisioningStatus':'String', 'appliesTo':'string'>> | No | None |
+| originalStart | datetime | The start time that was set when the event was created in UTC time. | No | None |
+| puser | string | User id. | No |   None |
+| ptenant | string |  Tenant id. | No  |  None |
+
+
+
+## JSON representation
+
+```json
+{  
+"id": "String (identifier)",
+"skuId": "Guid",
+"skuPartNumber": "String"
+"servicePlans": [{"@odata.type": "microsoft.graph.servicePlanInfo"}]
+} 
```

Comment:
Please add puser and ptenant in the json representation as well

Reply:
Added